ovos_number_parser/__init__.py

```
@@ -15,18 +15,20 @@
 from ovos_number_parser.numbers_eu import pronounce_number_eu, extract_number_eu, is_fractional_eu
 from ovos_number_parser.numbers_fa import pronounce_number_fa, extract_number_fa
 from ovos_number_parser.numbers_fr import (pronounce_number_fr, extract_number_fr, is_fractional_fr)
+from ovos_number_parser.numbers_gl import pronounce_number_gl, extract_number_gl, is_fractional_gl, numbers_to_digits_gl
 from ovos_number_parser.numbers_hu import pronounce_number_hu, pronounce_ordinal_hu
 from ovos_number_parser.numbers_it import (extract_number_it, pronounce_number_it, is_fractional_it)
 from ovos_number_parser.numbers_nl import numbers_to_digits_nl, pronounce_number_nl, pronounce_ordinal_nl, \
     extract_number_nl, is_fractional_nl
```

Ruff (0.12.2)

`ovos_number_parser.numbers_nl.is_fractional_nl` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)